mcs/class/System.Net.Http/monodroid_System.Net.Http.dll.sources

```diff
@@ -1,2 +1,2 @@
-#include System.Net.Http.dll.sources
+#include legacy.sources
```

Comment:
monotouch, monodroid and wasm have to use new version (it can be done in follow up PR)